microhapdb/cli/marker.py

```diff
@@ -85,8 +86,11 @@ def display(
             for marker in markers:
                 print(marker.detail)
         elif view_format == "fasta":
+            loci = defaultdict(Locus)
             for marker in markers:
-                print(marker.fasta)
+                loci[marker.locus].markers.append(marker)
+            for locus in loci.values():
+                print(locus.fasta)
```

Comment:
CLI update: store requested markers by locus, and then output the relevant locus sequences.